sideboard/tests/test_server.py

```diff
@@ -19,10 +18,15 @@
 
 import sideboard.websockets
 from sideboard.lib import log, config, subscribes, notifies, services, cached_property, WebSocket
-from sideboard.tests import service_patcher, config_patcher
+from sideboard.tests import service_patcher, config_patcher, get_available_port
 from sideboard.tests.test_sa import Session
 
 
+available_port = get_available_port()
+config['cherrypy']['server.socket_port'] = available_port
```

Comment:
I thought that might be the case. I guess we then still use our ConfigObj elsewhere?